### PR TITLE
Hotfix 23.10 - cudf, dask-cudf and cugraph

### PR DESCRIPTION
**Pull request by caryr35**
23.10 Hot-fix to correct cudf 23.10.1 that introduced a nasty regression that affects both dask_cudf=23.10 and cugraph=23.10

### CODE REVIEW

File: _notices/rgn0030.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Release Change
+notice_rapids_version: "v23.10"
+notice_created: 2023-11-16
```
Comment (aravenel) [inline _notices/rgn0030.md:22]:
```suggestion
notice_created: 2023-11-20
```

File: _notices/rgn0030.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-11-16
```
Comment (aravenel) [inline _notices/rgn0030.md:24]:
```suggestion
notice_updated: 2023-11-20
```

File: _notices/rgn0030.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+---
+
+## Overview
+
+`cudf 23.10.1` introduced a faulty regression that affects both `dask_cudf=23.10` and `cugraph=23.10`
```
Comment (aravenel) [inline _notices/rgn0030.md:29]:
```suggestion
`cudf 23.10.1` introduced a bug that could cause incorrect merge output in `cudf` and `dask-cudf` when merging multiple indexes where the index is not unique. `cudf` is being hotfixed to `23.10.2` to resolve this issue.
```

File: _notices/rgn0030.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## Rationale
+
+When there are duplicate values in an index, Series constructor shouldn't be reindexing if there is a new index passed.
```
Comment (aravenel) [inline _notices/rgn0030.md:33]:
```suggestion
`cudf 23.10.1` introduced a bug that could cause incorrect merge output in `cudf` and `dask-cudf` when merging multiple indexes where the index is not unique. `cudf` is being hotfixed to `23.10.2` to resolve this issue. The issue is not present in `cudf 23.12` nightlies.
```

File: _notices/rgn0030.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+We need a reindex fix(to fix cudf) and safe_hash fix (to fix dask_cudf) to correct the fautly regression issue
+
+## Impact
+
+All users are recommended to use the latest `23.12 Nightly Version of RAPIDS until hotfix is completed.
```
Comment (aravenel) [inline _notices/rgn0030.md:39]:
```suggestion
All `cudf` and `dask-cudf` users are encouraged to upgrade to the new `23.10.2` patch release.
```

File: _notices/rgn0030.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+## Overview
+
+`cudf 23.10.1` introduced a faulty regression that affects both `dask_cudf=23.10` and `cugraph=23.10`
+
+## Rationale
+
+When there are duplicate values in an index, Series constructor shouldn't be reindexing if there is a new index passed.
+
+We need a reindex fix(to fix cudf) and safe_hash fix (to fix dask_cudf) to correct the fautly regression issue
```
Comment (aravenel) [inline _notices/rgn0030.md:35]:
```suggestion
```

File: _notices/rgn0030.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+notice_topic: Release Change
+notice_rapids_version: "v23.10"
+notice_created: 2023-11-20
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-11-20
+---
+
+## Overview
+
+`cudf 23.10.1` introduced a bug that could cause incorrect merge output in `cudf` and `dask-cudf` when merging multiple indexes where the index is not unique. `cudf` is being hotfixed to `23.10.2` to resolve this issue.
```
Comment (beckernick) [inline _notices/rgn0030.md:29]:
```suggestion
`cudf 23.10.01` introduced a bug that could cause incorrect output in `cudf` and `dask-cudf` when merging dataframes in which one of the indexes is not unique. `cudf` is being hotfixed to `23.10.02` to resolve this issue.
```

File: _notices/rgn0030.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+## Rationale
+
+`cudf 23.10.1` introduced a bug that could cause incorrect merge output in `cudf` and `dask-cudf` when merging multiple indexes where the index is not unique. `cudf` is being hotfixed to `23.10.2` to resolve this issue. The issue is not present in `cudf 23.12` nightlies.
```
Comment (beckernick) [inline _notices/rgn0030.md:33]:
```suggestion
`cudf 23.10.01` introduced a bug that could cause incorrect output in `cudf` and `dask-cudf` when merging dataframes in which one of the indexes is not unique. `cudf` is being hotfixed to `23.10.02` to resolve this issue.
```

File: _notices/rgn0030.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+
+## Impact
+
+All `cudf` and `dask-cudf` users are encouraged to upgrade to the new `23.10.2` patch release.
```
Comment (beckernick) [inline _notices/rgn0030.md:38]:
```suggestion
All `cudf` and `dask-cudf` users are encouraged to upgrade to the new `23.10.02` patch release.
```

**Review by beckernick — APPROVED** [review]
Few minor suggestions, otherwise LGTM.

File: _notices/rgn0030.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rgn
+# Update meta-data for notice
+notice_id: 30 # should match notice number
+notice_pin: false # set to true to pin to notice page
+title: "RAPIDS v23.10 Hotfix for cuDF"
+notice_author: RAPIDS TPM
+notice_status: In Progress
+notice_status_color: yellow
```
Comment (beckernick) [inline _notices/rgn0030.md:13]:
```suggestion
notice_status: Completed
notice_status_color: green
```

**Review by aravenel — APPROVED** [review]

**Pull request merged**